web/courses.go

```diff
@@ -385,3 +385,17 @@ func (s *QuickFeedService) acceptRepositoryInvites(ctx context.Context, scmApp s
 	user.RefreshToken = newRefreshToken
 	return s.db.UpdateUser(user)
 }
+
+type orderMap map[uint64]uint32
+
+func newOrderMap(assignments []*qf.Assignment) *orderMap {
```

Comment:
This looks neat. Could you add a comment to explain what it does and also add a test with a few use-case examples?